LibraryApplication/LibraryApplication/Site.Master.cs

```diff
@@ -0,0 +1,43 @@
+﻿using LibraryApplication.Data;
+using System;
+using System.Web.Security;
+using System.Web.UI.WebControls;
+
+namespace LibraryApplication
+{
+    public partial class Site : System.Web.UI.MasterPage
+    {
+        private const string GetNameQuery = @"
+            SELECT FirstName + ' ' + LastName AS Name
+            FROM Employee
+            WHERE Username = @Username
+        ";
```

Comment:
The place to get additional user information is within the `Global.asax.cs` file so that you can pass that data into your `CustomPrincipal` class. Happy to walk through that process in person if you'd like.